helpers/integration_test.go

```diff
@@ -340,6 +341,38 @@ var _ = Describe("Pipeline Adapter", Ordered, func() {
 		Expect(result).To(BeFalse())
 	})
 
+	It("can handle broken json as HACBS_TEST_OUTPUT result", func() {
+		testpipelineRun.Status = tektonv1beta1.PipelineRunStatus{
+			PipelineRunStatusFields: tektonv1beta1.PipelineRunStatusFields{
+				TaskRuns: map[string]*tektonv1beta1.PipelineRunTaskRunStatus{
+					"task1": {
+						PipelineTaskName: "task-malformed-result",
+						Status: &tektonv1beta1.TaskRunStatus{
+							TaskRunStatusFields: tektonv1beta1.TaskRunStatusFields{
+								TaskRunResults: []tektonv1beta1.TaskRunResult{
+									{
+										Name:  "HACBS_TEST_OUTPUT",
+										Value: *tektonv1beta1.NewArrayOrString(`{
+											"success":false,
+											"errors":[{"code":6007,"message":"Malformed JSON in request body"}],
+											"messages":[],
+											"result":null
+										}`)
```

Comment:
Looks like a comma is missing here:
```
Error: helpers/integration_test.go:360:14: missing ',' before newline in composite literal
```

Reply:
Changed the formatting in a way I think removes some of the visual ambiguity here - I kept glancing right over this.